config/config.go

```diff
@@ -246,33 +229,38 @@ func loadConfiguration(file string) (*data.GortConfig, error) {
 	return &config, nil
 }
 
-// reloadConfiguration is called by both BeginChangeCheck() and Initialize()
-// to determine whether the config file has changed (or is new) and reload if
-// it has.
-func reloadConfiguration() error {
+// Reload is called by Initialize() to determine whether the config file has changed (or is new) and reload if it has.
+func Reload() error {
 	configMutex.Lock()
 	defer configMutex.Unlock()
 
 	sum, err := getMd5Sum(configFile)
 	if err != nil {
+		log.
+			WithField("file", configFile).
+			WithError(err).Error("Configuration reload failed")
```

Comment:
Is there other context we can add to these error messages to make it clear where the failure happened?

Reply:
The message is just commentary, really. The text of the actual error proper is emitted by the `WithError`. I can standardize the comment message a bit though.